frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/ChangeLogsPortletUtils.java

```diff
@@ -125,8 +130,12 @@ private JSONObject serveChangeLogsList(ResourceRequest request, ResourceResponse
             }
         }
 
-        List<ChangeLogs> changeLogsList = getFilteredChangeLogList(request, changeLogsClient);
+        List<ChangeLogs> changeLogsList = getFilteredChangeLogList(request, changeLogsClient, null);
```

Comment:
The getFilteredChangeLogList() function is used in 3 other places, so to avoid repeating the function
    I have added a check null condition in the getFilteredChangeLogList() function. 
    So that it can be used in other functions.